quay_setup.sh

```diff
@@ -1,4 +1,40 @@
 # oc create -f https://raw.githubusercontent.com/minio/minio-operator/master/minio-operator.yaml
+
+check(){
```

Comment:
I can't really take this as a solution. Too many hard coded assumptions.

Reply:
As discussed earlier.We cannot use the exit code to check whether quay operator is up.Because upon installing quay operator once and then even if you remove it."oc get QuayEcosystem" will exit with code 0.

Reply:
Also for checking whether the secret is populated or not.```oc get secrets quay-enterprise-config-secret -o json | jq -r '.data["config.yaml"]'``` returns 0 before the secret is populated and also after.